.github/ISSUE_TEMPLATE/apiary.yml

```diff
@@ -0,0 +1,22 @@
+name: Apiary
+description: Issue about documentation for ThePay.cz WS API
+labels: ["Apiary"]
+body:
+  - type: markdown
+    attributes:
+      value: "If you do not undestand something in ThePay WS API documentation, are you missing some information, or do you find the documentation inconsistent with values returned from the API, let us know in the description below."
```

Comment:
typo: If you don't understand something in ThePay WS API documentation, miss some information, or find the documentation inconsistent with values returned values from the API, let us know in the description below.

Reply:
Typo fixed, I looked in other templates and PhpStorm spellcheck not found any other typos like this.

Reply:
There is still just a little mistake: `values returned values` fix to: `returned values`, sorry.

Reply:
np

Reply:
fixed